sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+  AtomXmlSerializer,
+  serializeToAtomXmlRequest,
+  deserializeAtomXmlResponse,
+  HttpOperationResponse
+} from "@azure/core-http";
+import * as Constants from "../util/constants";
+
+export interface QueueOptions {
+  /**
+   * Determines the amount of time in seconds in which a message should be locked for processing by a receiver. After this period, the message is unlocked and available for consumption by the next receiver. Settable only at queue creation time.
+   */
+  LockDuration?: string;
```

Comment:
We should be using camel casing to be consistent with how we represent interface members across the libraries we own in this repo.

Reply:
The interface describes the properties which the user directly sets - these have to be in the exact casing else they don't get set by the service.

Reply:
Does the service expect the properties to be in this exact casing in the http request?

Reply:
Resolving as we know that service expects pascal casing.
As per latest changes, we take input in camel casing from the user, do the conversion and then send request